model/course.go

```diff
@@ -68,6 +69,54 @@ func (c *Course) SetCameraPresetPreference(pref []CameraPresetPreference) {
 	c.CameraPresetPreferences = string(pBytes)
 }
 
+type SourcePreference struct {
+	LectureHallID uint `json:"lecture_hall_id"`
+	SourceMode    int  `json:"source_mode"`
```

Comment:
Can you create a type for the source mode like here: https://github.com/joschahenningsen/TUM-Live/blob/15f794a088386c5fbf5f1246e14f05cb9edaa0f2/model/stream.go#L16

(this is usually how we simulate mocks in go)

Reply:
Done